src/VerifyWindowsHelloController.js

```diff
@@ -94,22 +113,88 @@ function (Okta, FormController, FormType, webauthn, FooterSignout, WindowsHelloE
             )
           );
         }
+        else {
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-default">\
+              {{i18n code="verify.windowsHello.subtitle" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: null}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-loading">\
+              {{i18n code="verify.windowsHello.subtitle.loading" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'loading'}
+          }));
+
+          result.push(FormType.View({
+            View: '\
+            <p class="okta-form-subtitle o-form-explain text-align-c" data-se="o-form-subtitle-windows-hello-success">\
+              {{i18n code="verify.windowsHello.subtitle.signingIn" bundle="login"}}\
+            </p>',
+            showWhen: {__enrollmentState__: 'success'}
+          }));
+        }
+
+        result.push(FormType.View({View: Spinner}));
+
         return result;
       },
 
+      postRender: function () {
+        if (this.options.appState.get('factors').length === 1 && !this.model.get('__enrollmentState__')) {
+          this.model.save();
```

Comment:
can you tell me what is this ```postRender``` for? Is it to show Windows Hello modal right away?
If so, this should not be only when windows is the only factor, but instead it should be when windows hello is the selected factor (does not matter if there are more factors or not)
@vincentvoong-okta is this correct?